pkg/printers/internalversion/printers_test.go

```diff
@@ -829,29 +829,19 @@ func TestPrintNodeStatus(t *testing.T) {
 			node: api.Node{
 				ObjectMeta: metav1.ObjectMeta{
 					Name:   "foo10",
-					Labels: map[string]string{"node-role.kubernetes.io/master": "", "kubernetes.io/role": "node", "kubeadm.alpha.kubernetes.io/role": "node"},
+					Labels: map[string]string{"node-role.kubernetes.io/master": "", "node-role.kubernetes.io/proxy": "", "kubernetes.io/role": "node"},
 				},
 			},
-			status: "Unknown,Master",
+			status: "master,node,proxy",
```

Comment:
@liggitt Seems `TestPrintNodeStatus` did not find out the bug that `Role` is not represented with `Status`.  

```Golang
if !contains(strings.Fields(buffer.String()), test.status) {
	t.Fatalf("Expect printing node %s with status %#v, got: %#v", test.node.Name, test.status, buffer.String())
}
```

This check is not strictly matched, shall we use `strings.Fields(buffer.String())[2]` instead of this. Of course we will make sure there is no index out of range.

If there is a consensus, I would like to submit a separate PR fixing this.